src/FSharp.Data.GraphQL.Server.Middleware/SchemaDefinitions.fs

```diff
@@ -88,11 +88,15 @@ module SchemaDefinitions =
         | :? ObjectListFilter as x -> Some x
         | _ -> None
 
+    let serialize (x : ObjectListFilter) : Value =
+        StringValue (string x) // TODO
```

Comment:
What do you mean by TODO?

Reply:
This `serialize` implementation here has the old behavior (just stringify the object), which is broken (strictly speaking). However, the sample works as before.

Reply:
So what should be ideally done here?